static/test/javascripts/conf/adblock.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (config) {
```

Comment:
You can run an individual Jasmine unit test by changing the name of the describe block to `fdescribe` - the 'f' standing for 'focus'. The advantage of this approach is that you can never accidentally commit it - there is a linting rule - run on pushing upstream - that will bounce tests with `fdescribe` and `fit` blocks.